Other/ExtractingPartitions.md

```diff
@@ -18,44 +18,31 @@
 
 ## Getting your current boot slot (A/B)
 
-Boot to Android™, on your PC open a command prompt window and type:
+Assuming your Surface Duo is booted to Android™, plugged into to your PC:
 
 ```
-adb reboot fastboot
+adb reboot bootloader
 ```
 
-Your Surface Duo will be rebooted to fastbootd (not fastboot). Once you're there:
+Your Surface Duo will reboot into fastboot. Once you're there:
 
 ```
-fastboot getvar all
+fastboot getvar current-slot
 ```
 
-A lot of text lines will be printed. Look for a line that says `(bootloader) current-slot:`. It can be a or b. In my case:
-
-<img width="204" alt="image" src="https://user-images.githubusercontent.com/29689637/222553048-6d473cbe-820a-452d-a77c-5b42cbd2682b.png">
-
-Take note of your slot, we'll need that later. For the rest of this guide we'll assume it is `b`.
+A line with the text 'current-slot:' will appear.
+The value can be a or b.
 
-Now let's get back into Android™:
-
-```
-fastboot reboot
-```
+<img width="304" alt="image" src="https://github-production-user-asset-6210df.s3.amazonaws.com/75797743/242037668-45949c27-b4bc-4abb-90a9-b5a4060ba648.png">
 
-## Extracting the boot or other partitions
+Take note of your slot, we'll need that later.
```

Comment:
adding "we'll assume it is 'b'" helps solves ambiguity further down the line, otherwise users wouldn't know what to replace, here everytime they would see _b they would know. Maybe there's a need for a better refactoring. A lot of the guide work here is a bit like doing social engineering in real life, except you're dealing with people that unfortunately have the tendency to read steps way too fast or outright skip any that they don't understand :P